### DISCUSSION TITLE
Capture of Korath World Ships

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#8349] [Q&A] [answered]
Any tips on successfully capturing Korath World Ships? I don't know of any ships that have over 500 capacity apart from the World Ships, and currently cannot stand a chance against them.

**Comment by Luuiscool4567** [comment]
What version are you playing on?

**Reply by SultanRTG** [reply]
9.16.1

**Comment by Hecter94** [comment, marked as answer]
A couple of things to consider: 
1) For capturing, you want to maximize your capture attack while minimizing capture defense. Generally this means you want to use nerve gas and only nerve gas. Sell anything you have that isn't nerve gas. 
2) There's a good amount of variance in how much crew a worldship can spawn with, attacking ones with low crew will give you an advantage. You can use a tactical scanner to see crew. 
3) Make sure the worldship you capture is either in a hyperjump accessible location, or has a jump drive, else you won't be able to keep it. 
4) Finally, make sure to maximize your crew on your flagship and let your escorts handle the fighting. 

If you follow those instructions, you should be able to capture one. Good luck.

**Reply by SultanRTG** [reply]
I got the worldship, but it did not have a jump drive and now it is stuck in exile space. Whoops!

**Reply by Naf5000** [reply]
Wait, why minimize capture defense? I was under the impression that that increased the survivability of your crew, while capture attack increases their lethality.

**Reply by Hecter94** [reply]
When you go to attack a ship, the game lets you either attack or defend, and the AI also chooses whether to attack or defend.
If you both defend, you retreat.
If you attack, and the AI defends, it compares your total attack rating versus the AI's total defense rating.
If you attack and the AI attacks, it compares your total attack rating versus the AI's total attack rating

Since the vast majority of ships have far more defense than they do attack, you really want the AI to attack you so that you fight against their much lower attack score compared to your attack score. 
So you minimize your defense to ensure that the AI chooses to attack you, and then maximize your attack to ensure you can overpower their attack. 

Think of it as making your ship appear purposefully weak to lure them into attacking so you can best them with your superior attack.

**Reply by MasterOfGrey** [reply]
> Wait, why minimize capture defense? I was under the impression that that increased the survivability of your crew, while capture attack increases their lethality.

Basically, nothing currently works the way that you'd intuitively expect it to.
(We have a project for reworking that but certain devs haven't been available to look at it properly.)

**Comment by movingpictures** [comment]
fyi, it juuuuuuust so happens that the world ships with jump drives also happen to have a lot of crew and are thus uncapturable.

**Reply by SultanRTG** [reply]
In theory, if you have enough explosives and chemicals...

**Comment by Danekjovax** [comment]
I have played this game for a few months now, and have at this point accumulated 5 Korath world ships.  All of them are the 577-type.  It is MUCH HARDER to capture any of the other types, either because of the ludicrously high crew counts and/or their superior defenses (microbots FTW).

But, for all practical purposes, the 577 is about as versatile as any ship you'll need to use in the mid-late game, so I'll give a brief description on how I got my first one:

Before starting, make sure you're already getting missions and news from other space port feeds about some "alien ships" being spotted in the syndicate space.  This usually happens after you're done with the Free Worlds campaign, but perhaps it can trigger independent of it.  

1) I first started with a Kimek Spire, maxxed out its crew bunk to something around 320-ish (I don't have the exact config, but it was around that), and also picked up some nerve gas (enough for the whole crew) and a tactical scanner (any type will do) while in Misam or in Polaris (pirate systems at the east end of Syndicate space bordering the Korath exile systems).  Dominating those systems will make hanging out in them a lot less costly and cumbersome, so make sure you bring in a sufficient fleet (I use at least 8 or so Dreadnaught/Shield Beetle size ships to start) to get them to get them bend the knee.
2) First, I max out the crew on my "boarding flagship", then I stay in either of the aforementioned systems (I prefer Misam since the pirate base also includes a shipyard/outfitter) with my fleet and clear out any pirates (making note of any sizeable ones for later - DO NOT board them right now) and wait for a large blip to warp in.  That's likely going to be a Palavret (Raider) or Rag'no'erek (Dredger).  You'll want the latter but either will do (the Raider will make Dredger boarding a little easier, but you can still take a Dredger with a Spire if you're patient).  Once you've disabled the ship, use your tactical scanner to view the crew compliment of your target.  Dredgers with less than 250 crew should be beatable, so only board when you see that number or less.  The first one I took had around 230 or so and I only had a 35% chance to win, but I got lucky so I got my first Dredger early on.  I use these as they're better for boarding than the Spire (I can usually get double the bunks on a Dredger!).
3) Haul it into the pirate base and get it maxed out with bunks, nerve gas, (I also include a tactical scanner and at least one point defense turret and one attack turret).  Make sure its got sufficient shield/armor regen (usually the system core it's already got is sufficient but sometimes you may need to upgrade to a large version).  Fill your crew, get your fleet (at this point have at least a dozen Dreadnaught or larger set of ships - the bigger and tankier the better, and equip them with continuous-beam weapons with heat damage if possible), and jump into Korath space for your first "fishing" expedition!
4) I prefer going to the northmost Korath system (the one northmost of the "triangle" of systems) and positioning my escorts to the north of the Korath station that's there.  Make sure you position your fleet in a horizontal line formation (clicking on your ships individually and right-clicking an empty space will make them move to that location and hold position there) and put them far enough away that the regular system-jumping Korath ships don't automatically "kite" in pursuit of your formed fleet.  Back them up northwards if that happens.  The ships you want to grab will occasionally jump in, and they will jump in the space between your fleet and the station, and given your fleet's proximity at that location will kite them after your fleet, which should quickly dispatch and disable them for your picking.

Most of the ships that will be caught in this "trap" will be mostly Raiders, with the occasional Dredger and some of the JD-equipped world ships (618 and 621 models) which are unfortunately out of reach of our current boarding capacity.  That's ok.  Patience is the game winner here.  Eventually a 577-type with a JD equipped will jump into range and engage your fleet.  That's the quarry you're looking for.

5) After a few attempts at the rare 577 jumping in and being disabled, you should eventually find one with a low enough compliment (230-250) that you should be able to take them with your boarding flagship.

Haul it back to your pirate forward base and celebrate!

Cheers, and happy "fishing"! ;2)

**Reply by SultanRTG** [reply]
I've already got a dredger and pretty strong ships (Dredger with ~400 crew capacity and 2 Arfectas, with ~11 Hurricanes and other ships that I forgot about). I can also buy some more ships (open to suggestions for this). Which weapons are better when you are facing a lower crew-Korath Detainers or Nerve gas?

**Reply by Danekjovax** [reply]
I used Nerve Gas for any boarding action, at least until I was able to access Bunrodea's Blasters (rated 2.6) or the Heliarch's Enforcer Confrontation Gear (3.1) - both "legal" outfits.
